### PR TITLE
Eric 2

### PR DESCRIPTION
**Pull request by ericmnelson**

### CODE REVIEW

File: final_project/year_app/year_app.py [hunk 1]
```diff
@@ -1,24 +1,67 @@
-from flask import Flask, render_template
+from flask import Flask, request, session, url_for, redirect, \
+     render_template, abort, g, flash, _app_ctx_stack
 import numpy as np
 import pickle
+import requests
+import pprint
+pp = pprint.PrettyPrinter(indent=4)
+
+from pyechonest import config
```
Comment (maxwolffe) [inline final_project/year_app/year_app.py]: Do you need this config stuff in this file?

Comment (ericmnelson) [inline final_project/year_app/year_app.py]: What config stuff? the pyechonest config?

Comment (maxwolffe) [inline final_project/year_app/year_app.py]: Nvm. I see that you do. Maybe we can just move all of our echonest api requests into the echo_req.py file?

Comment (ericmnelson) [inline final_project/year_app/year_app.py]: Yea we can figure out best structure for the code later I was just trying to hack together a proof of concept.

**Comment by maxwolffe** [comment]
This is sweet. LGTM! I'll add instructions for installing the echonest API on our readme.

**Pull request merged**